src/src/components/App/index.tsx

```diff
@@ -2,10 +2,27 @@ import React from 'react';
 import { commonText } from '../../localization/common';
 import { useBooleanState } from '../../hooks/useBooleanState';
 import { Portal } from '../Molecules/Portal';
+import { useEffect } from "react";
+
 
 export function App() {
   const [isOpen, _, handleClose, handleToggle] = useBooleanState();
 
+  useEffect(() => {
```

Comment:
(nonblocking) Totally optional:

I try to use `React.useEffect(` rather than `useEffect(` for default React hooks. This helps differentiate them from custom hooks (like `useBooleanState`)